kolibri/plugins/management/assets/src/vue/class-edit-page/index.vue

```diff
@@ -23,18 +23,17 @@
     </div>
 
     <div class="toolbar">
-      <div class="searchbar" role="search">
+      <div class="searchbar" :class="{'searchbar-mobile': elSize.width < 600}" role="search">
```

Comment:
Thank you for switching to the mixin!

For this case, it's more appropriate to use the `responsive-window` plugin and the pre-defined breakpoints so that we're not hard-coding values.

In particular, 

```javascript
elSize.width < 600
```

can be replaced  with:

```javascript
windowSize.breakpoint <= 1
```

If you look at the docs in _responsive-window.js_ you'll see the various definitions of the breakpoint levels.

In general:
* use responsive-window in full-size pages
* use responsive-element in smaller components that don't take up the full window width